Description: Fix FTBFS with GCC 11
 The error message was: "static assertion failed: comparison object
 must be invocable as const"
Author: Axel Beckert <abe@debian.org>
Forwarded: Tim Day <timday@timday.com>
Bug-Debian: https://bugs.debian.org/984054

--- a/libevolvotron/mutatable_image_computer_farm.h
+++ b/libevolvotron/mutatable_image_computer_farm.h
@@ -45,7 +45,7 @@
     {
     public:
       //! Compare task priorities.
-      bool operator()(const boost::shared_ptr<const MutatableImageComputerTask>& t0,const boost::shared_ptr<const MutatableImageComputerTask>& t1) 
+      bool operator()(const boost::shared_ptr<const MutatableImageComputerTask>& t0,const boost::shared_ptr<const MutatableImageComputerTask>& t1)  const
 	{ 
 	  return (t0->priority() < t1->priority());
 	}
@@ -56,7 +56,7 @@
     {
     public:
       //! Compare task priorities.
-      bool operator()(const boost::shared_ptr<const MutatableImageComputerTask>& t0,const boost::shared_ptr<const MutatableImageComputerTask>& t1) 
+      bool operator()(const boost::shared_ptr<const MutatableImageComputerTask>& t0,const boost::shared_ptr<const MutatableImageComputerTask>& t1) const
 	{ 
 	  return (t0->priority() > t1->priority());
 	}
